code/game/gamemodes/nuclear/nuclearbomb.dm

```diff
@@ -342,6 +342,8 @@ GLOBAL_VAR(bomb_set)
 /obj/machinery/nuclearbomb/attack_hand(mob/user as mob)
```

Comment:
Nitpick, but do we still need `as mob` here? I thought it was mostly deprecated except for some things like verbs?

Reply:
Not really needed, but I want really want to expand the scope of this PR

Reply:
Fair, feel free to keep it in then, fine to stay for a future PR